app/components/footer_component.html.erb

```diff
@@ -62,7 +62,7 @@
             <li><a class="link--black" href="/train-to-be-a-teacher/if-you-dont-have-a-degree">Train to teach without a degree</a></li>
             <li><a class="link--black" href="/train-to-be-a-teacher/teacher-degree-apprenticeships">Teacher degree apprenticeship</a></li>
             <li><a class="link--black" href="/train-to-be-a-teacher/assessment-only-route-to-qts">Assessment only QTS</a></li>
-            <li><a class="link--black" href="/life-as-a-teacher/teaching-as-a-career/change-careers">Retrain as a teacher</a></li>
+            <li><a class="link--black" href="/life-as-a-teacher/change-careers">Retrain as a teacher</a></li>
```

Comment:
Link fine - I wonder whether should move this up ahead of TDA now we have more content?